src/main/java/racingcar/model/Race.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.model;
+
+import racingcar.domain.Cars;
+
+import java.util.Random;
+
+public class Race {
+    private static final int RANDOM_BOUND = 10;
+    private static final int RACE_THRESHOLD = 4;
+
+    private final Random random;
+
+    public Race() {
+        this.random = new Random();
+    }
+
+    public void roundRace(Cars cars) {
+        cars.getCarList().forEach(car -> {
+            int randomValue = random.nextInt(RANDOM_BOUND);
+            if (randomValue > RACE_THRESHOLD) car.move();
+        });
+    }
```

Comment:
```suggestion
    public void roundRace(Cars cars) {
        cars.getCarList().forEach(car -> {
            int randomValue = random.nextInt(RANDOM_BOUND);
            if (randomValue > RACE_THRESHOLD) {
                car.move();
            }
        });
    }
```
지난 step과 다르게 car 구현이 달라졌는데 변경하신 이유가 있을까요? 🤔
현재는 car의 움직임을 객체 스스로가 판단하고 있지 않고 외부에서 제어하고 있어서요.